HELPFILE

```diff
@@ -1,44 +1,80 @@
+                                section of the Codecov report. Useful to help
+                                resolve path fixing                     [string]
+  -n, --name                    Custom defined name of the upload. Visible in
+                                Codecov UI                [string] [default: ""]
+  -N, --parent                  The commit SHA of the parent for which you are
+                                uploading coverage. If not present, the parent
+                                will be determined using the API of your
+                                repository provider. When using the repository
+                                provider's API, the parent is determined via
+                                finding the closest ancestor to the commit.
+                                                                        [string]
+  -P, --pr                      Specify the pull request number manually[number]
+  -Q, --source                  Used internally by Codecov, this argument helps
+                                track
+                                wrappers of the uploader (e.g. GitHub Action,
+                                CircleCI Orb)             [string] [default: ""]
+  -R, --rootDir                 Specify the project root directory when not in a
+                                git repo
+  -r, --slug                    Specify the slug manually [string] [default: ""]
+  -s, --dir                     Directory to search for coverage reports.
+                                Already searches project root and current
+                                working directory                       [string]
+  -T, --tag                     Specify the git tag       [string] [default: ""]
+  -t, --token                   Codecov upload token      [string] [default: ""]
+  -U, --upstream                The upstream http proxy server to connect
+                                through                   [string] [default: ""]
+  -u, --url                     Change the upload host (Enterprise use)
+                                        [string] [default: "https://codecov.io"]
+  -v, --verbose                 Run with verbose logging               [boolean]
+  -X, --feature                 Toggle functionalities. Separate multiple ones
+                                by comma: -X network,search
+                                -X fixes         Enable file fixes to ignore
+                                common lines from coverage (e.g. blank lines or
+                                empty brackets)
+                                -X network       Disable uploading the file
+                                network
+                                -X search        Disable searching for coverage
+                                files                                   [string]
+      --xcode, --xc, --xc       Run with xcode support[boolean] [default: false]
```

Comment:
FWIW, looks like `--xc` is listed twice:
https://github.com/codecov/uploader/blob/cb97e5f75b4bba3afed0b07380d5ab29c34d66be/src/helpers/cli.ts#L204-L210
https://github.com/codecov/uploader/blob/cb97e5f75b4bba3afed0b07380d5ab29c34d66be/src/helpers/cli.ts#L231-L237